Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2.py

```diff
@@ -1569,6 +1695,10 @@ def search_records_command(client: Client, args: dict[str, str]):
     )
     level_id = args.get("levelId")
 
+    xml_filter_condition = args.get("xmlForFiltering")
+    if xml_filter_condition and not is_valid_xml(xml_filter_condition):
```

Comment:
```suggestion
    if (xml_filter_condition := args.get("xmlForFiltering")) and not is_valid_xml(xml_filter_condition):
```

Reply:
Fixed in commit cbeb616e474a7c5d67a4e15ac24b85286fcb58c8